EKF/control.cpp

```diff
@@ -931,8 +931,11 @@ void Ekf::controlHeightFusion()
 	checkRangeAidSuitability();
 	_range_aid_mode_selected = (_params.range_aid == 1) && isRangeAidSuitable();
 
-	if (_params.vdist_sensor_type == VDIST_SENSOR_BARO) {
-
+	switch (_params.vdist_sensor_type) {
+	default:
```

Comment:
What is the default case added for? Why the fallthrough?

Reply:
I thought it couldn't hurt if someone entered a bogus HGT_MODE.